subsys/usb/usb_descriptor.c

```diff
@@ -88,7 +88,7 @@ USBD_DEVICE_DESCR_DEFINE(primary) struct common_descriptor common_desc = {
 		.bConfigurationValue = 1,
 		.iConfiguration = 0,
 		.bmAttributes = USB_CONFIGURATION_ATTRIBUTES,
-		.bMaxPower = MAX_LOW_POWER,
+		.bMaxPower = CONFIG_USB_MAX_POWER,
```

Comment:
please do the same for `bMaxPower = MAX_LOW_POWER` in subsys/usb/class/usb_dfu.c
please remove `#define MAX_LOW_POWER                  0x32` from include/usb/usb_common.h
```diff
diff --git a/include/usb/usb_common.h b/include/usb/usb_common.h
index 632524a464..5fa617e098 100644
--- a/include/usb/usb_common.h
+++ b/include/usb/usb_common.h
@@ -86,11 +86,6 @@
 #define BCDDEVICE_RELNUM               (BCD(KERNEL_VERSION_MAJOR) << 8 | \
                                        BCD(KERNEL_VERSION_MINOR))
 
-/* 100mA max power, per 2mA units */
-/* USB 1.1 spec indicates 100mA(max) per unit load, up to 5 loads */
-#define MAX_LOW_POWER                  0x32
-#define MAX_HIGH_POWER                 0xFA
-
 /* Highest value of Frame Number in SOF packets. */
 #define USB_SOF_MAX                    2047
 
diff --git a/subsys/usb/class/usb_dfu.c b/subsys/usb/class/usb_dfu.c
index bc3169f0e0..750c5c936f 100644
--- a/subsys/usb/class/usb_dfu.c
+++ b/subsys/usb/class/usb_dfu.c
@@ -156,7 +156,7 @@ struct dev_dfu_mode_descriptor dfu_mode_desc = {
                .bConfigurationValue = 1,
                .iConfiguration = 0,
                .bmAttributes = USB_CONFIGURATION_ATTRIBUTES,
-               .bMaxPower = MAX_LOW_POWER,
+               .bMaxPower = CONFIG_USB_MAX_POWER,
        },
        .sec_dfu_cfg = {
                /* Interface descriptor */

```

Reply:
will do.